### ISSUE TITLE
Not enough information returned from Webhook 'ANNOTATION_DELETED'

### ISSUE THREAD

**Issue by svecile** [HumanSignal/label-studio-sdk#305]
label-studio-sdk version: 1.0.5, label studio version: 1.13.1 (Docker deployment)

I'm using webhooks to keep an external waveform dataset up to date. However, the only information returned from the webhook event 'ANNOTATION_DELETED' is the annotation_id but i need some way to figure out which task the annotation belonged to so i can remove the corresponding labels from the external dataset. Since the annotation (and its ID) have been deleted i have no way to query label studio to figure out which task it belonged to. When you get an annotation_created or annotation_updated event you get the task information along with it and in the 'data' field i can put information such as the start and end time of the section of waveform the annotations belong to. This allows me to query the external database and update labels accordingly. 

In short there a way to figure out which task or task_id a deleted annotation belonged to so i can get the task info by querying the tasks.get endpoint?
